holidays/countries/chile.py

```diff
@@ -23,6 +22,12 @@
 from holidays.constants import TUE, WED, THU, FRI, SAT, SUN
 from holidays.holiday_base import HolidayBase
 
+# use standard library for timezone
+try:
+    from zoneinfo import ZoneInfo
+except ImportError:  # pragma: no cover
```

Comment:
It's not a coverage improvement, it's a coverage decrease suppression :)

I had a similar problem in [this PR](https://github.com/dr-prodigy/python-holidays/pull/889/). It took me a while to figure out a better solution. I'll take care of the refactoring after #889 is merged.

Reply:
Yeah, I agree, it's definitely a suppression.

I can't think how to do it "correctly", so I look forward to seeing what you come up with! :)

Reply:
You can find an example in https://github.com/dr-prodigy/python-holidays/pull/889, I'm going to use pretty much the same approach.